.travis.yml

```diff
@@ -1,16 +1,20 @@
-langauge: ruby
+language: ruby
+cache: bundler
 rbenv:
 - 2.5.3
+before_install:
+- gem install bundler
 addons:
   postgresql: 9.6
   chrome: stable
-dist: trusty
+dist: xenial
```

Comment:
trusty and chrome aren't compatible (which is apparently a recent thing)

Reply:
this looks like a lot of googling and troubleshooting